components/list/list-header.js

```diff
@@ -35,14 +42,36 @@ class ListHeader extends RtlMixin(LocalizeCoreElement(LitElement)) {
 			 * TODO: Remove
 			 * @type {boolean}
 			 */
-			slim: { reflect: true, type: Boolean }
+			slim: { reflect: true, type: Boolean },
+			_scrolled: { type: Boolean, reflect: true }
 		};
 	}
 
 	static get styles() {
 		return css`
 			:host {
+				background-color: var(--d2l-list-header-background-color, white);
 				display: block;
+				position: sticky;
+				top: 0;
+				z-index: 6; /* must be greater than d2l-list-item-active-border */
+			}
+			:host([no-sticky]) {
+				background-color: transparent;
+				position: static;
+				z-index: auto;
+			}
+			.d2l-list-header-shadow {
+				transition: box-shadow 200ms ease-out;
+			}
+			:host([_scrolled]) .d2l-list-header-shadow {
```

Comment:
This fancy header shadow is used for a couple reasons - to cover up whitespace that is reserved for the drop target indicator; and also to have a fancy drop-shadow that fades at the edges so that it looks good when the header does not go edge to edge (ex. when consumer container has padding).